Makefile

```diff
@@ -34,8 +37,8 @@ FIND_EXT_ := $(foreach EXT,$(EXTENSIONS),-o -name "*.$(EXT)")
 FIND_EXT := $(wordlist 2,$(words $(FIND_EXT_)),$(FIND_EXT_))
 FIND_IGNORE := $(foreach IG,$(IGNORE),! -path "./$(IG)*")
 
-SOURCE_FILES := $(subst ./,,$(shell find . \( $(FIND_EXT) \) $(FIND_IGNORE) -exec test -e {} \; -print))
-JSON_SOURCE_FILES := $(subst ./,,$(shell find . -name "*.json" $(FIND_IGNORE) -exec test -e {} \; -print))
+SOURCE_FILES := $(subst ./,,$(shell find $(SOURCE) \( $(FIND_EXT) \) $(FIND_IGNORE) -exec test -e {} \; -print))
```

Comment:
I'm fine with this, but we'll need to change the `subst` call in front as well. Something like `$(subst $(SOURCE)/,,…` so that the prefix is stripped off correctly.